README.md

```diff
@@ -46,6 +46,23 @@ To make Discord's launch options persistent, add them to `~/.var/app/com.discord
 --password-store=basic
 ```
 
+### Opening Discord automatically at system startup
+
+The Discord app has a built-in option for that, but it doesn't work on Flatpak mostly because of sandbox restrictions.
+
+But we can still do that manually, with these two commands:
+
+```sh
+mkdir -p ~/.config/autostart
```

Comment:
I know this should technically be `${XDG_CONFIG_HOME:-"$HOME/.config"}`, but I kept it simple on purpose.

Besides, anyone actually defining `XDG_CONFIG_HOME` most likely knows what they're doing. :nerd_face: